src/components/NavBar/PlatformsMenu.tsx

```diff
@@ -1,33 +1,36 @@
-import './platforms.css';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { platforms } from '../../assets/platforms';
+import PlatformsStyle from './Platforms.module.css';
+import RoundButtonStyle from '../BasicComponents/RoundButton.module.css';
```

Comment:
why not import the round button ? ( if you need additional style- send props to the fellow)